### PR TITLE
:ambulance: Unavailability to return fractional product amount

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
Sometimes fractional values appear, such as 4.440892098500626e-16
image

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
It is possible to return more than purchased.
image

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

File: pos_orders_history_return/doc/changelog.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.. _changelog:
+`1.0.6`
+-------
 
-Updates
-=======
+**Fix:** Unavailability to return fractional product amount
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history_return/doc/changelog.rst:4]: :+1: checked

File: pos_orders_history_return/static/src/js/models.js [hunk 2]
```diff
@@ -53,7 +57,7 @@ odoo.define('pos_orders_history_return.models', function (require) {
             }
             var el = $('article[data-product-id="'+product.id+'"] .max-return-qty');
             var qty = this.get_current_product_return_qty(product);
-            el.html(product.max_return_qty - qty);
+            el.html(Math.round((product.max_return_qty - qty) * 1000) / 1000);
```
Comment (yelizariev) [inline pos_orders_history_return/static/src/js/models.js]: use odoo tools instead: round_decimals, round_precision

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by em230418 — CHANGES_REQUESTED** [review]
Тест с этим модулем автоматически не запускаются, т.к. в неподходящее имя файла в котором определен тест - default_test.py. Название модуля должна начинаться с test_

Пруфлинк: https://github.com/odoo/odoo/blob/a22419e45124aa578785853efdb98db79c9ae65c/doc/reference/testing.rst#testing-python-code

File: pos_orders_history_return/static/src/js/models.js (hunk 2 above)
Comment (KolushovAlexandr) [reply pos_orders_history_return/static/src/js/models.js]: I especially changed them to Math round due to they do not round properly
https://github.com/it-projects-llc/pos-addons/compare/0c216a8129fcef17f32319e567026a49a2cafe3e..dce6bf77ebb57014c65723ae01d3412efa8a71a7
Previous Ramils reviews were fixed by not using them

**Comment by em230418** [comment]
Сам tour обламывается на `.control-button.orders-history`

File: pos_orders_history_return/static/src/js/models.js (hunk 2 above)
Comment (yelizariev) [reply pos_orders_history_return/static/src/js/models.js]: You can use proper number, instead of variable this.pos.currency.rounding

**Review by em230418 — APPROVED** [review]

**Review by yelizariev — APPROVED** [review]

**Pull request merged**